### PR TITLE
Add re-sft scripts

### PR DESCRIPTION
**Pull request by so298**
```
max_seq_length 4096
bf16
lr_scheduler_type cosine
warmup_ratio 0.1
learning_rate 2e-5 or 1e-4
global_batch_size = gradient_accumulation_steps * n_gpu * per_device_train_batch_size = 64
```
で設定

出力先は`/model/7B_HF_RE/*`

- lora-all-jaster
- lora-all-jaster-lr_1e-4
- lora-all-gpt4-self-inst
- lora-all-gpt4-self-inst-lr_1e-4

を追加

### CODE REVIEW

**Comment by so298** [comment]
exp2とexp3を追加

**Comment by so298** [comment]
exp6とexp8のgpt4-self-instのデータセットに関してはsingle-GPU学習だとメモリ不足で学習ができなかったので、1 node 8 GPUを利用した学習へと変更

$\text{global batch size} = N_\text{GPUs} \times \text{per device batch size} \times \text{gradient accumulation steps} = 64$
となるように`per_device_batch_size = 1, gradient_accumulation_steps = 8`に調整

**Comment by so298** [comment]
@llm-jp/modelwg 
draftのまま放置してしまっていました
今更ですが、誰かreviewとマージを行っていただけるでしょうか？

**Comment by hiroshi-matsuda-rit** [comment]
明日以降、私の方でレビューを行いたいと思いますが、実験の状況を理解していないものが多くあるので、いろいろ確認を入れさせてもらうかもしれません。ご了承ください。
